### ISSUE TITLE
BCH stuck in "Unverified" status

### ISSUE THREAD

**Issue by TexasMiner** [fyookball/electrum#385]
I recently switched from mining BTC to mining BCH at Antpool.  My payments (BCH) all show as going out as normal (as it did with bitcoin), but they never show up in my "Electon Cash" wallet.  Every now and then I will see (during a sync I guess) that the transactions are showing up as "unverified" deposits but disappear shortly after they appear.  Can you offer any explanation as to what is going on?  Why don't the "unverified" deposits show continuously on in the "Electron Cash" history? I am attaching a screenshot that I managed to capture one time before it disappeared.  I will also include below the transactions.  Thank you for any help you can offer.
d798c8abb7dd32e4f626a2a0c26aadd40fc9e8f953cd61e39bf17eb118190b47
049e3e4136c479b8f2ff508e4e726dfe2b3110b2d7c6f0f03e3a4b5cf68625bd
7500eb308d015f83e9508477f0e21f7345806d0721df0d99346d272c53ca0f80
52108788ca536e36d673e0c35c6b971c240acb2f3aca9db2e38ac3294170626f
363b94cc8ef34906faeeccd62de0986ac769945e66a89736def298adfd5e9720
5fe10995f72dba93a6937f873b9eee61f43b6ee8482ccaa84dbc0ecb75bc5b49
635e142411d022d151f5a1f1461e081d02c052b603d5a0616f0f582cae4e07c6

![unverified deposits bch](https://user-images.githubusercontent.com/34838853/34346201-5442aa46-e9c3-11e7-8d29-1a07d5da2ff9.JPG)

**Comment by ryankemper** [comment]
@TexasMiner I don't have a lot of experience mining myself, so I may be unable to help you, but I see noone else has replied. Are you still experiencing the issue? I loaded the first txid you posted,  d798c8abb7dd32e4f626a2a0c26aadd40fc9e8f953cd61e39bf17eb118190b47, and I am seeing >500 confirmations, so the coins should be yours. Also what version of electron cash are you using?

https://blockchair.com/bitcoin-cash/transaction/d798c8abb7dd32e4f626a2a0c26aadd40fc9e8f953cd61e39bf17eb118190b47

**Comment by TexasMiner** [comment]
Yes, I have seen that also.  When I switched my wallet to Jaxx, the deposits hit as normal.  I have used antpool and electrum  for bitcoin for a long time, but when I switched it to BCH at antpool and used electron cash as my wallet, I never saw a deposit.  the only way I ever even see the "unverified" as seen below, is when I select electron-ABC-cryptolayer.net as the server.  then I at least see the "unverfied" transactions.  otherwise I never even see that. If I put the server on autoselect, the "unverifieds" dissapear from view.   It is a bit of money so I feel it is worth pursuing.    I also included my son's "Electron wallet".  it did the same thing.  We have used the wallet for other things before but this is not working.  Thanks for your input and concern.

![unverified0 deposits bch](https://user-images.githubusercontent.com/34838853/34428537-d7f99322-ec1b-11e7-9b87-63c72ed69465.JPG)
